recipes/cheat/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "cheat" %}
+{% set version = "2.1.26" %}
+{% set sha256 = "f5f6d86e7c1c3eb6c595156b534bc2a0305bd3a3433e20e0e78952a356aa347a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install
+  skip: True  # [win and py3k]
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python
+    - docopt >=0.6.1
+    - pygments >=1.6.0
+
+test:
+  imports:
+    - cheat
+    - cheat.cheatsheets
+    - cheat.test
+
+about:
+  home: https://github.com/chrisallenlane/cheat
+  license: GPL3
+  # license_file: LICENSE # When added to MANIFEST.in – chrisallenlane/cheat#291
+  license_family: GPL
```

Comment:
So I guess this must be `GPL3`. 😖 Alternatively I'm ok with it being dropped as it is redundant.